customerbalancetransaction.go

```diff
@@ -22,8 +30,8 @@ const (
 // For more details see https://stripe.com/docs/api/customers/create_customer_balance_transaction
 type CustomerBalanceTransactionParams struct {
 	Params      `form:"*"`
+	Customer    *string `form:"-"` // Included in URL
```

Comment:
That isn't in alphabetical anymore. Not the end of the world but is it something we can easily tweak?

Reply:
The convention now is embeds, path params, then normal params, and alphabetical within those categories. A saw the hoisting of path params in a couple files and thought it made sense even though we don't follow it consistently, but yes I can easily alphabetize the whole thing if we think that is better.

Maybe a newline between the path params and the remainder?

Reply:
I personally prefer everyone alphabetical. What's URL param and whats POST param shouldn't really matter to developers and it's easier for people's brain (and ours) to parse alphabetically. Not a blocker though.